src/decima/plot/visualize.py

```diff
@@ -201,18 +192,44 @@ def plot_marker_box(
     return g
 
 
-def plot_attribution_peaks(attr, tss_pos):
+def plot_peaks(attrs, tss_pos, df_peaks=None, overlapping_min_dist=1000, figsize=(10, 2)):
+    """Plot peaks in attribution scores.
+
+    Args:
+        attr: Attribution scores array
+        tss_pos: Position of TSS
```

Comment:
We should specify "Relative position of TSS in the interval" to make it clear this is not the genomic coordinate